File: semester2/programming/list3/zad1/Kolo.java
import java.lang.Math;

public class Kolo implements Figura {
    public double obliczObwod()
    {
        return 2 * Math.PI * radius;
    }
    public double obliczPole()
    {
        return Math.PI * radius * radius;
    }
    public Kolo(double _radius)
    {
        radius = _radius;
    }

    private double radius;
}

File: semester2/programming/list3/zad2/Kolo.cpp
#define _USE_MATH_DEFINES
#include "Kolo.hpp"


Kolo::Kolo(double _radius) : radius(_radius) {}


double Kolo::obliczPole()
{
    return M_PI * radius * radius;
}
double Kolo::obliczObwod()
{
    return 2 * M_PI * radius;
}